flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -155,6 +156,10 @@
 
 	private String zookeeperNamespace;
 
+	// Pre-uploaded flink files, key is relative path(for example, lib/flink-dist_{version}.jar),
+	// and value is remote file status.
+	private final Map<String, FileStatus> preUploadedFlinkFiles;
```

Comment:
nit: follow the other configs pattern, i.e., init a empty map and call `addAll` when deploy instead of initializing in constructor. It can be a follow-up we generally refactor deployment inside YarnClusterDescriptor.

Reply:
Will fix it.